pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/internal/pex/PexExecOutputParser.java

```diff
@@ -15,14 +15,14 @@
  */
 package com.linkedin.gradle.python.util.internal.pex;
 
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
 import org.gradle.api.GradleException;
 import org.gradle.api.logging.Logger;
 import org.gradle.api.logging.Logging;
 import org.gradle.process.ExecResult;
 
+import java.util.regex.Matcher;
```

Comment:
Again, I don't see the need for this change.
We can talk again about import order in case I'm misunderstanding something.

Ditto for the next file, except, in it, an empty line should be moved below java.util.Map import. So, perhaps it's OK to change that one.

There are other places like this. That's why I'm not commenting on them, until we figure out the intent.